--- a/resources/scripts/components/dashboard/AccountOverviewContainer.tsx
+++ b/resources/scripts/components/dashboard/AccountOverviewContainer.tsx
@@ -37,6 +37,7 @@
                 </MessageBox>
             )}
 
+            <div id="notur-slot-account.overview.before" style={{ display: 'contents' }}></div>
             <Container css={[tw`lg:grid lg:grid-cols-3 mb-10`, state?.twoFactorRedirect ? tw`mt-4` : tw`mt-10`]}>
                 <ContentBox title={'Update Password'} showFlashes={'account:password'}>
                     <UpdatePasswordForm />
@@ -48,6 +49,7 @@
                     <ConfigureTwoFactorForm />
                 </ContentBox>
             </Container>
+            <div id="notur-slot-account.overview.after" style={{ display: 'contents' }}></div>
         </PageContentBlock>
     );
 };
