--- a/resources/scripts/components/server/backups/BackupContainer.tsx
+++ b/resources/scripts/components/server/backups/BackupContainer.tsx
@@ -35,7 +35,6 @@
     return (
         <ServerContentBlock title={'Backups'}>
             <FlashMessageRender byKey={'backups'} css={tw`mb-4`} />
-            <div id="notur-slot-server.backups.before" style={{ display: 'contents' }}></div>
             <Pagination data={backups} onPageSelect={setPage}>
                 {({ items }) =>
                     !items.length ? (
@@ -72,7 +71,6 @@
                     )}
                 </div>
             </Can>
-            <div id="notur-slot-server.backups.after" style={{ display: 'contents' }}></div>
         </ServerContentBlock>
     );
 };
