--- a/resources/scripts/components/server/databases/DatabasesContainer.tsx
+++ b/resources/scripts/components/server/databases/DatabasesContainer.tsx
@@ -39,6 +39,7 @@
     return (
         <ServerContentBlock title={'Databases'}>
             <FlashMessageRender byKey={'databases'} css={tw`mb-4`} />
+            <div id="notur-slot-server.databases.before" style={{ display: 'contents' }}></div>
             {!databases.length && loading ? (
                 <Spinner size={'large'} centered />
             ) : (
@@ -75,6 +76,7 @@
                     </>
                 </Fade>
             )}
+            <div id="notur-slot-server.databases.after" style={{ display: 'contents' }}></div>
         </ServerContentBlock>
     );
 };
