--- a/resources/scripts/components/server/network/NetworkContainer.tsx
+++ b/resources/scripts/components/server/network/NetworkContainer.tsx
@@ -52,7 +52,6 @@
 
     return (
         <ServerContentBlock showFlashKey={'server:network'} title={'Network'}>
-            <div id="notur-slot-server.network.before" style={{ display: 'contents' }}></div>
             {!data ? (
                 <Spinner size={'large'} centered />
             ) : (
@@ -78,7 +77,6 @@
                     )}
                 </>
             )}
-            <div id="notur-slot-server.network.after" style={{ display: 'contents' }}></div>
         </ServerContentBlock>
     );
 };
