--- a/resources/scripts/routers/ServerRouter.tsx
+++ b/resources/scripts/routers/ServerRouter.tsx
@@ -20,7 +20,6 @@
 import ConflictStateRenderer from '@/components/server/ConflictStateRenderer';
 import PermissionRoute from '@/components/elements/PermissionRoute';
 import routes from '@/routers/routes';
-import { getNoturRoutes } from '@/routers/routes';
 
 export default () => {
     const match = useRouteMatch<{ id: string }>();
@@ -77,7 +76,6 @@
                     <CSSTransition timeout={150} classNames={'fade'} appear in>
                         <SubNavigation>
                             <div>
-                                <div id="notur-slot-server.subnav.before" style={{ display: 'contents' }}></div>
                                 {routes.server
                                     .filter((route) => !!route.name)
                                     .map((route) =>
@@ -99,10 +97,7 @@
                                         <FontAwesomeIcon icon={faExternalLinkAlt} />
                                     </a>
                                 )}
-                                <div id="notur-slot-server.subnav" style={{ display: 'contents' }}></div>
-                                <div id="notur-slot-server.subnav.after" style={{ display: 'contents' }}></div>
                             </div>
-                            <div id="notur-slot-server.header" style={{ display: 'contents' }}></div>
                         </SubNavigation>
                     </CSSTransition>
                     <InstallListener />
@@ -121,16 +116,11 @@
                                             </Spinner.Suspense>
                                         </PermissionRoute>
                                     ))}
-                                    {/* [Notur] Extension server routes */}
-                                    {getNoturRoutes('server').map((route: any, i: number) => (
-                                        <Route key={`notur-server-${i}`} {...route} />
-                                    ))}
                                     <Route path={'*'} component={NotFound} />
                                 </Switch>
                             </TransitionRouter>
                         </ErrorBoundary>
                     )}
-                    <div id="notur-slot-server.footer" style={{ display: 'contents' }}></div>
                 </>
             )}
         </React.Fragment>
