--- a/resources/scripts/components/server/startup/StartupContainer.tsx
+++ b/resources/scripts/components/server/startup/StartupContainer.tsx
@@ -84,7 +84,6 @@
         )
     ) : (
         <ServerContentBlock title={'Startup Settings'} showFlashKey={'startup:image'}>
-            <div id="notur-slot-server.startup.before" style={{ display: 'contents' }}></div>
             <div css={tw`md:flex`}>
                 <TitledGreyBox title={'Startup Command'} css={tw`flex-1`}>
                     <div css={tw`px-1 py-2`}>
@@ -131,7 +130,6 @@
                     <VariableBox key={variable.envVariable} variable={variable} />
                 ))}
             </div>
-            <div id="notur-slot-server.startup.after" style={{ display: 'contents' }}></div>
         </ServerContentBlock>
     );
 };
