--- a/resources/scripts/components/server/databases/DatabasesContainer.tsx
+++ b/resources/scripts/components/server/databases/DatabasesContainer.tsx
@@ -39,7 +39,6 @@
     return (
         <ServerContentBlock title={'Databases'}>
             <FlashMessageRender byKey={'databases'} css={tw`mb-4`} />
-            <div id="notur-slot-server.databases.before" style={{ display: 'contents' }}></div>
             {!databases.length && loading ? (
                 <Spinner size={'large'} centered />
             ) : (
@@ -76,7 +75,6 @@
                     </>
                 </Fade>
             )}
-            <div id="notur-slot-server.databases.after" style={{ display: 'contents' }}></div>
         </ServerContentBlock>
     );
 };
