--- a/resources/scripts/components/NavigationBar.tsx
+++ b/resources/scripts/components/NavigationBar.tsx
@@ -58,8 +58,12 @@
                     >
                         {name}
                     </Link>
+                    {/* [Notur] Navbar left extension slot */}
+                    <div id="notur-slot-navbar.left" style={{ display: 'contents' }}></div>
                 </div>
                 <RightNavigation className={'flex h-full items-center justify-center'}>
+                    {/* [Notur] Navbar before extension slot */}
+                    <div id="notur-slot-navbar.before" style={{ display: 'contents' }}></div>
                     <SearchContainer />
                     <Tooltip placement={'bottom'} content={'Dashboard'}>
                         <NavLink to={'/'} exact>
@@ -73,6 +77,8 @@
                             </a>
                         </Tooltip>
                     )}
+                    {/* [Notur] Navbar extension slot */}
+                    <div id="notur-slot-navbar" style={{ display: 'contents' }}></div>
                     <Tooltip placement={'bottom'} content={'Account Settings'}>
                         <NavLink to={'/account'}>
                             <span className={'flex items-center w-5 h-5'}>
@@ -85,6 +91,8 @@
                             <FontAwesomeIcon icon={faSignOutAlt} />
                         </button>
                     </Tooltip>
+                    {/* [Notur] Navbar after extension slot */}
+                    <div id="notur-slot-navbar.after" style={{ display: 'contents' }}></div>
                 </RightNavigation>
             </div>
         </div>
