--- a/resources/scripts/components/server/settings/SettingsContainer.tsx
+++ b/resources/scripts/components/server/settings/SettingsContainer.tsx
@@ -25,6 +25,7 @@
     return (
         <ServerContentBlock title={'Settings'}>
             <FlashMessageRender byKey={'settings'} css={tw`mb-4`} />
+            <div id="notur-slot-server.settings.before" style={{ display: 'contents' }}></div>
             <div css={tw`md:flex`}>
                 <div css={tw`w-full md:flex-1 md:mr-10`}>
                     <Can action={'file.sftp'}>
@@ -81,6 +82,7 @@
                     </Can>
                 </div>
             </div>
+            <div id="notur-slot-server.settings.after" style={{ display: 'contents' }}></div>
         </ServerContentBlock>
     );
 };
