--- a/resources/scripts/components/server/users/UsersContainer.tsx
+++ b/resources/scripts/components/server/users/UsersContainer.tsx
@@ -50,7 +50,6 @@
     return (
         <ServerContentBlock title={'Users'}>
             <FlashMessageRender byKey={'users'} css={tw`mb-4`} />
-            <div id="notur-slot-server.users.before" style={{ display: 'contents' }}></div>
             {!subusers.length ? (
                 <p css={tw`text-center text-sm text-neutral-300`}>It looks like you don&apos;t have any subusers.</p>
             ) : (
@@ -61,7 +60,6 @@
                     <AddSubuserButton />
                 </div>
             </Can>
-            <div id="notur-slot-server.users.after" style={{ display: 'contents' }}></div>
         </ServerContentBlock>
     );
 };
