--- a/resources/views/layouts/admin.blade.php	2026-02-03 20:34:23
+++ b/resources/views/layouts/admin.blade.php	2026-02-03 20:34:23
@@ -85,6 +85,14 @@
                                 <i class="fa fa-gamepad"></i> <span>Application API</span>
                             </a>
                         </li>
+                        @if (Route::has('admin.notur.extensions'))
+                            <li class="header">EXTENSIONS</li>
+                            <li class="{{ ! starts_with(Route::currentRouteName(), 'admin.notur') ?: 'active' }}">
+                                <a href="{{ route('admin.notur.extensions') }}">
+                                    <i class="fa fa-puzzle-piece"></i> <span>Extensions</span>
+                                </a>
+                            </li>
+                        @endif
                         <li class="header">MANAGEMENT</li>
                         <li class="{{ ! starts_with(Route::currentRouteName(), 'admin.databases') ?: 'active' }}">
                             <a href="{{ route('admin.databases') }}">
