--- a/resources/scripts/components/server/console/Console.tsx
+++ b/resources/scripts/components/server/console/Console.tsx
@@ -221,7 +221,6 @@
                     >
                         <ChevronDoubleRightIcon className={'w-4 h-4'} />
                     </div>
-                    <div id="notur-slot-server.console.command" style={{ display: 'contents' }}></div>
                 </div>
             )}
         </div>
