--- a/resources/scripts/components/dashboard/DashboardContainer.tsx	2026-02-03 20:41:36
+++ b/resources/scripts/components/dashboard/DashboardContainer.tsx	2026-02-03 20:41:36
@@ -50,6 +50,7 @@

     return (
         <PageContentBlock title={'Dashboard'} showFlashKey={'dashboard'}>
+            <div id="notur-slot-dashboard.header" style={{ display: 'contents' }}></div>
             {rootAdmin && (
                 <div css={tw`mb-2 flex justify-end items-center`}>
                     <p css={tw`uppercase text-xs text-neutral-400 mr-2`}>
@@ -66,21 +67,26 @@
                 <Spinner centered size={'large'} />
             ) : (
                 <Pagination data={servers} onPageSelect={setPage}>
-                    {({ items }) =>
-                        items.length > 0 ? (
-                            items.map((server, index) => (
-                                <ServerRow key={server.uuid} server={server} css={index > 0 ? tw`mt-2` : undefined} />
-                            ))
-                        ) : (
-                            <p css={tw`text-center text-sm text-neutral-400`}>
-                                {showOnlyAdmin
-                                    ? 'There are no other servers to display.'
-                                    : 'There are no servers associated with your account.'}
-                            </p>
-                        )
-                    }
+                    {({ items }) => (
+                        <>
+                            <div id="notur-slot-dashboard.serverlist.before" style={{ display: 'contents' }}></div>
+                            {items.length > 0 ? (
+                                items.map((server, index) => (
+                                    <ServerRow key={server.uuid} server={server} css={index > 0 ? tw`mt-2` : undefined} />
+                                ))
+                            ) : (
+                                <p css={tw`text-center text-sm text-neutral-400`}>
+                                    {showOnlyAdmin
+                                        ? 'There are no other servers to display.'
+                                        : 'There are no servers associated with your account.'}
+                                </p>
+                            )}
+                            <div id="notur-slot-dashboard.serverlist.after" style={{ display: 'contents' }}></div>
+                        </>
+                    )}
                 </Pagination>
             )}
+            <div id="notur-slot-dashboard.footer" style={{ display: 'contents' }}></div>
         </PageContentBlock>
     );
 };
