--- a/resources/scripts/components/server/schedules/ScheduleEditContainer.tsx
+++ b/resources/scripts/components/server/schedules/ScheduleEditContainer.tsx
@@ -80,7 +80,6 @@
     return (
         <PageContentBlock title={'Schedules'}>
             <FlashMessageRender byKey={'schedules'} css={tw`mb-4`} />
-            <div id="notur-slot-server.schedules.edit.before" style={{ display: 'contents' }}></div>
             {!schedule || isLoading ? (
                 <Spinner size={'large'} centered />
             ) : (
@@ -169,7 +168,6 @@
                     </div>
                 </>
             )}
-            <div id="notur-slot-server.schedules.edit.after" style={{ display: 'contents' }}></div>
         </PageContentBlock>
     );
 };
